modules/webgl2-polyfill/src/polyfills/multidraw-polyfill.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable max-params */
+
+export function multiDrawArraysPolyfill(
```

Comment:
as far as I can see, these extensions can be polyfilled client side (without performance benefits of course)...

TBD - is this the right thing to do?